gui/default/index.html

```diff
@@ -329,9 +332,39 @@ <h3 class="panel-title">
         </div>
       </div>
 
-      <!-- First regular row -->
+      <!-- Login form -->
+      <div ng-if="!authenticated" class="center-block">
+        <h3 translate>Authentication required</h3>
 
-      <div class="row">
+        <form ng-submit="authenticatePassword()">
+          <div class="form-group">
+            <label for="Username" translate>Username</label>
+            <input id="Username" class="form-control" type="text" ng-model="login.username" />
+          </div>
+
+          <div class="form-group">
+            <label for="Password" translate>Password</label>
+            <input id="Password" class="form-control" type="password" ng-model="login.password" ng-trim="false" />
+          </div>
+
+          <div class="row">
+            <div class="col-md-9 login-form-messages">
+              <p ng-if="login.errors.badLogin" class="text-danger">
+                Incorrect username and/or password.
```

Comment:
```suggestion
                Incorrect user name or password.
```

I'd also suggest to use just "or" instead of "and/or" (the latter is legalese).